plugins/kubernetes/app/views/samson_kubernetes/_build_show.erb

```diff
@@ -0,0 +1,4 @@
+<div class="form-group">
+  <%= check_box_tag :kubernetes_job%>
+  <%= form.label :kubernetes_job, "Use Remote Kubernetes Job", class: "control-label" %>
```

Comment:
Sounds good to me. Only one small issue is to switch between local build and remote build option without an UI. What is your suggestion?